src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { get, map } from 'lodash';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { selectCurrentlyOpenEditModal } from '../selectors';
+import { openModal, createTransaction } from '../actions';
+
+import * as VET360 from '../constants';
+
+class AddressValidationModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedOption: '',
+      address: {},
+    };
+  }
+  onChangeHandler = address => event => {
+    this.setState({ selectedOption: event.target.name, address });
+  };
+
+  onSubmit = () => {
+    const { validationKey } = this.props;
+    const { address } = this.state;
+    const payload = { ...address, validationKey };
+
+    const method = payload.id ? 'PUT' : 'POST';
+
+    this.props.createTransaction(
+      VET360.API_ROUTES.EMAILS,
+      method,
+      this.props.fieldName,
+      payload,
+      this.props.analyticsSectionName,
+    );
+  };
+
+  renderWarningText = () => {
+    const { addressValidationError } = this.props;
+
+    return addressValidationError
+      ? "We're sorry. It looks like the address you entered isn't valid.  Please edit your address"
+      : "We're sorry. It looks like the address you entered isn't valid.  Please enter your address again or choose a suggested address below";
+  };
+
+  renderPrimaryButton = () => {
+    const {
+      addressValidationError,
+      addressValidationType,
+      validationKey,
+    } = this.props;
+
+    if (addressValidationError && !!validationKey) {
+      return (
+        <button
+          className="usa-button-parimary"
+          onClick={() => this.props.openModal(addressValidationType)}
+        >
+          Edit Address
+        </button>
+      );
+    }
+
+    return <button className="usa-button-parimary">Continue</button>;
+  };
+
+  renderAddressOption = (address, id = 'userEntered') => {
+    const { validationKey } = this.props;
+    const {
+      addressLine1,
+      addressLine2,
+      addressLine3,
+      city,
+      stateCode,
+      zipCode,
+    } = address;
+
+    const isOptionDisabled = id === 'userEntered' && !!validationKey;
+
+    return (
+      <div key={id}>
+        <input
+          style={{ zIndex: '1' }}
+          type="radio"
+          name={id}
+          disabled={isOptionDisabled}
+          checked={this.state.selectedOption === id}
+          onClick={this.onChangeHandler(address)}
+        />
+        <label
+          htmlFor={id}
+          style={{
```

Comment:
We have utility classes to use instead of writing inline styles. https://design.va.gov/utilities/flexbox 

So in this case you could add the following: `className="vads-u-display--flex vads-u-align-items--center"`

Reply:
Awesome, thanks for that.  I was wondering where those lived.